docs/content/docs/guide/repositorycrd.md

```diff
@@ -398,3 +398,32 @@ and the scoping fails for the repository level configuration because the reposit
   ```yaml
   failed to scope github token as repo owner5/project5 does not exist in namespace test-repo
   ```
+
+### Don't automatically allow the users from the organisation to start Pipelines-as-Code on a Repository
+
+By default, when using for example the GitHub provider, if your repository is
+belong to an organization, the users belonging to that organization
+are granted automatic permissions to initiate the pipelines. However,
+considering the potential existence of malicious users within a large organization
+and the need to exercise caution, there may be certain repositories for which we
+do not wish to extend trust to everyone.
+
+To address this, you have the option to deactivate this functionality by
+configuring the repository settings and setting the
+`only_trusts_users_from_repository` parameter to `true`.
+
+```yaml
+apiVersion: "pipelinesascode.tekton.dev/v1alpha1"
+kind: Repository
+metadata:
+  name: test
+  namespace: test-repo
+spec:
+  url: "https://github.com/linda/project"
+  settings:
+    only_trusts_users_from_repository: true
```

Comment:
How about `trusts_users_from_repository` :thinking:

Reply:
yeah it took me a long time to find a name...

if we say trusts_users_from_repository it's like we would not trust users from repository if that setting is not set which is not true....

Reply:
`trust_users_from_repository_only` ?